site/content/index.md

```diff
@@ -119,16 +119,56 @@ layout: unstyled
   <div className="prose dark:prose-invert mx-auto max-w-2xl px-4 lg:max-w-6xl lg:px-8 xl:px-12">
     <h2 className="text-center">Features</h2>
 
-    Here are some of the cool Flowershow features:
 
-    - CommonMark, GitHub Flavoured Markdown and Obsidian extensions support (e.g. wikilinks) [[syntax|➡ learn more]]
-    - MDX support for custom React components additions [[custom-components|➡ learn more]]
-    - Customizable layouts of your pages [[layouts|➡ learn more]]
-    - Tailwind support for easy styling in-markdown HTML and custom components [[tailwind|➡ learn more]]
-    - Google analytics support [[analytics|➡ learn more]]
-    - Basic site configurations using single JSON file [[basic-config|➡ learn more]]
+Here are some of the cool features that are currently supported by Flowershow, and some that we're intensively working on.
 
-    👷 There is a lot of stuff we're still working on.
+### ✅/🚧 Markdown
+Flowershow supports **CommonMark** and **GitHub Flavoured Markdown (GFM)** syntax, e.g. code blocks, blockquotes, lists, tasks lists and tables.
+
+[[syntax|Learn more ➡]]
+
+### ✅/🚧  Obsidian extensions to Markdown
+Flowershow has been designed with Obsidian users in mind, and so it aims to support the whole suite of Obsidian syntax extensions, like:
+- internal links ✅,
+- callouts 🚧,
+- math 🚧,
```

Comment:
I'd say Math and Mermaid aren't obsidian extensions but rather complete features in their own right with their own section. I'd also say that for callouts in fact.

I'd add as obsidian syntax the transclusion stuff that we don't fully support yet i.e. `![[...]]` type syntax.

So i'd call this section Obsidian wiki links and put it as ✅ or 🚧 depending on how important transclusion is.

Reply:
I've rearranged it into: 

### ✅ Obsidian wiki links
and
### 🚧 Excalidraw, Mermaid an Mathjax support

Reply:
OK. Though i'd not that each of those is a feature in itself! 😄

Reply:
I wanted to keep it concise on the front page so I've listed them under one heading, but I can split it if you wish :slightly_smiling_face: